src/ostree/ot-builtin-commit.c

```diff
@@ -111,6 +112,7 @@ static GOptionEntry options[] = {
   { "owner-uid", 0, 0, G_OPTION_ARG_INT, &opt_owner_uid, "Set file ownership user id", "UID" },
   { "owner-gid", 0, 0, G_OPTION_ARG_INT, &opt_owner_gid, "Set file ownership group id", "GID" },
   { "canonical-permissions", 0, 0, G_OPTION_ARG_NONE, &opt_canonical_permissions, "Canonicalize permissions in the same way bare-user does for hardlinked files", NULL },
+  { "mode-w-xor-x", 0, 0, G_OPTION_ARG_NONE, &opt_mode_w_xor_x, "Ensure executable files are not writable", NULL },
```

Comment:
Hmm, that's a bit cryptic (also I think it's technically a nand, not a xor :) ). Maybe something like `--ro-executables`?

Reply:
"xor" is about the "rule" and not the operation - https://en.wikipedia.org/wiki/W%5EX

Reply:
Ahh gotcha, didn't realize that.